test/end-to-end/indexing.test.ts

```
@@ -1159,4 +1160,119 @@ describe('Indexing', function () {
+        BUNDLE_REPAIR_UPDATE_TIMESTAMPS_INTERVAL_SECONDS: '1',
+      });
+
+      bundlesDb = new Sqlite(`${projectRootPath}/data/sqlite/bundles.db`);
+    });
+
+    afterEach(async function () {
+      bundlesDb.close();
+      await compose.down();
+    });
+
+    it.only('should reset last_fully_indexed_at when filters change', async function () {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)